source/stylesheets/modules/_toc.scss

```diff
@@ -44,3 +44,7 @@
     display: none;
   }
 }
+
+.toc-link--in-view {
+  font-weight: bold;
```

Comment:
This was requested by Dave House – apparently he's keen to have it match the behaviour in the service manual.